test/CoWSwapEthFlow.t.sol

```diff
@@ -421,3 +422,79 @@ contract OrderDeletion is EthFlowTestSetup {
         vm.stopPrank();
     }
 }
+
+contract SignatureVerification is EthFlowTestSetup {
+    bytes4 internal constant BAD_SIGNATURE = 0xffffffff;
+
+    function dummyOrder() internal view returns (EthFlowOrder.Data memory) {
```

Comment:
Why don't you you the existing function dummyOrder?

Reply:
I wanted to use different values to make sure there wasn't some issue related to the fact that we use (more or less) one single order example in the tests. I'm definitely open to reuse `dummyOrder` however.

Reply:
> Note that you can have "floating" Solidity functions in recent compiler version (may be easier to share "floating" functions than having contracts).

I didn't know about this, thanks for sharing! [Here](https://blog.soliditylang.org/2020/09/02/solidity-0.7.1-release-announcement/) is a description of how they work, they can only be pure functions and importing a file that contains them imports all free functions in the current scope.